schema/pgsql/upgrades/017.sql

```diff
@@ -0,0 +1,18 @@
+TRUNCATE TABLE rule_escalation_recipient;
```

Comment:
Please try to avoid truncating tables if possible, especially if it's manually entered configuration. And in the next block, you're writing statements to actually migrate from `channel_type` to `channel_id`, so if it can be done for one table, it should be easy to do for all.